pandas/core/groupby/generic.py

```diff
@@ -77,11 +77,8 @@
 from pandas.core.series import Series
 from pandas.core.util.numba_ import (
     NUMBA_FUNC_CACHE,
```

Comment:
can you remove the CACHE from here as well?

Reply:
There are some separate code paths that store the function in the cache independently

Reply:
i c. ideally this is not visible at all in this code if at all possible (this is what I mean by using a Dispatcher, e.g. that you would be how you would interactive with the function & cache itself). but i guess that's for the future.

Reply:
Ah I see. Okay I can look into that in a future PR when playing around more with a Dispatcher